infra/data-pipeline.yml

```diff
@@ -56,6 +56,34 @@ tasks:
       owner: release-mgmt-analysis@mozilla.com
       source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
 
+  - ID: rollback-test-task
+    created: {$fromNow: ''}
+    deadline: {$fromNow: '25 hours'}
+    provisionerId: aws-provisioner-v1
+    workerType: releng-svc
+    dependencies:
+      - bugs-retrieval
+    payload:
+      maxRunTime: 3600
+      image: python
+      command:
+        - "/bin/bash"
+        - "-lcx"
+        - "git clone ${repository} &&
+           cd bugbug &&
+           git checkout ${head_rev} &&
```

Comment:
Related to the prev comment, head_rev is undefined here, so it wouldn't know what to checkout to. If we use the base image, this should be fixed automatically